kt/godot-library/src/main/kotlin/godot/core/bridge/VariantArray.kt

```diff
@@ -207,15 +207,15 @@ class VariantArray<T> : NativeCoreType, MutableCollection<T> {
      *
      * **Note**: For an empty array, this method always returns `true`.
      */
-    fun all(callable: Callable): Boolean {
+    fun all(callable: NativeCallable): Boolean {
```

Comment:
Should still be Callable here, otherwise users won't be able to use a Kotlin lambda alongside a VariantArray.